ReactWindows/ReactNative/Bridge/JavaScriptBundleLoader.cs

```diff
@@ -1,179 +0,0 @@
-﻿using System;
-using System.Threading.Tasks;
-using Windows.Storage;
-using static System.FormattableString;
-
-namespace ReactNative.Bridge
-{
```

Comment:
Can you double check if squashing these commits makes this a file rename instead of a delete and add?